.gitmodules

```diff
@@ -5,6 +5,3 @@
 	path = _data/navbars
 	url = https://github.com/ioos/documentation-theme-jekyll
 	branch = navbars
-[submodule "_wiki"]
-	path = _wiki
-	url = https://github.com/ioos/ioosngdac.wiki.git
```

Comment:
The wiki page was here as a submodule. I guess that folks tried to use them in the docs at some point.